cachi2/core/package_managers/generic.py

```diff
@@ -1,13 +1,54 @@
-from cachi2.core.errors import PackageRejected
+import logging
+from typing import Literal, Optional
+
+import yaml
+from pydantic import BaseModel, ValidationError, field_validator
+
+from cachi2.core.errors import PackageManagerError, PackageRejected
 from cachi2.core.models.input import Request
 from cachi2.core.models.output import RequestOutput
 from cachi2.core.models.sbom import Component
 from cachi2.core.rooted_path import RootedPath
 
+LOG = logging.getLogger(__name__)
```

Comment:
nitpick: we usually define this as a lowercase variable.

Reply:
renamed